components/common/Progressbar.tsx

```diff
@@ -10,37 +9,20 @@ const ProgressWrapper = styled('div', {
 const Progress = styled('div', {
   height: '100%',
   backgroundColor: '$textPrimary',
-  variants: {
-    size: {
-      1: {
-        width: '20%',
-      },
-      2: {
-        width: '40%',
-      },
-      3: {
-        width: '60%',
-      },
-      4: {
-        width: '80%',
-      },
-      5: {
-        width: '100%',
-      },
-    },
-  },
 });
 
-type ProgressVariantProps = VariantProps<typeof Progress>;
-
 type ProgressbarProps = {
   value: number;
 };
 
 export const Progressbar = ({ value }: ProgressbarProps) => {
   return (
     <ProgressWrapper>
-      <Progress size={value as ProgressVariantProps['size']} />
+      <Progress
+        style={{
+          width: `${value * 20}%`,
```

Comment:
이전 코드를 통해 유추해볼때, 20~100% 사이의 값만 적용되길 원하신다면 [css clamp()](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp) 함수를 쓰기 좋은 케이스인것 같아요.

```suggestion
          width: `clamp(20%, ${value * 20}%, 100%)`,
```